### PR TITLE
Adding integration test for a participant without a domain of experti…

### PR DESCRIPTION
**Pull request by mmilitaru**
Integration test for validation of a domain for a participant when participant is updated.

### CODE REVIEW

File: IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "domain": [ "0e92edde-fdff-41db-9b1d-f2e484f12535" ]
+    }
+  ],
+  "_create": [],
+  "_update": [
+    {
+      "classKind": "Participant",
+      "iid": "284334dd-e8e5-42d6-bc8a-715c507a7f02",
+      "modifiedOn": "2020-04-14T17:42:59.264Z",
+      "selectedDomain": "00000000-0000-0000-0000-000000000000"
```
Comment (lxatrhea) [inline IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json:15]: The value here `"00000000-0000-0000-0000-000000000000"` is not an empty value. 
It should be `null`

File: IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿{
+  "_delete": [
```
Comment (lxatrhea) [inline IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json:2]: Do we need a delete operation here?

File: IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json (hunk 1 above)
Comment (mmilitaru) [reply IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json:15]: We do not use Nullable<Guid> so, Guid is Not Nullable always comes as empty.

**Comment by mmilitaru** [comment]
Sam, please let us have a chat tomorrow. As, I have went deep into the issue, including database inspection, I do not believe that a participant can exist without a domain assignment. In db, the Selected domain/assigned domains are not nullable, and in relationship with domain of expertise, so they never can be missing or misplaced. Also, the selected domain of a participant is a Guid and not a nullable Guid, so even the participant object cannot set the selected domain to null. As I said let us have a skype tomorrow to make sure we are on the same page. I hope I did not miss anything.

File: IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/ParticipantTestFixture.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ParticipantTestFixture.cs" company="RHEA System">
+//
+//   Copyright 2016 RHEA System
```
Comment (lxatrhea) [inline IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/ParticipantTestFixture.cs:4]: Please update copyright notice

File: IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    {
+      "classKind": "Participant",
+      "iid": "284334dd-e8e5-42d6-bc8a-715c507a7f02",
+      "domain": null,
+      "modifiedOn": "2020-04-14T17:42:59.264Z",
+      "selectedDomain": null
+    }
+  ],
+  "_copy": []
+}
```
Comment (lxatrhea) [inline IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/Participant/Post_Participant_With_Null_Domain.json:14]: Please add a newline at the bottom of the file.

**Comment by samatrhea** [comment]
closed in 202b3c399efd4f8481063c1cb763cd704828858c

**Pull request closed without merging**